environments/ithc/ithc.tfvars

```
@@ -45,6 +45,29 @@ frontends = [
       },
     ]
   },
+  {
+    name             = "em-showcase"
+    custom_domain    = "em-showcase-ithc.service.core-compute-ithc.internal"
+    backend_domain   = ["firewall-nonprodi-palo-ithc.uksouth.cloudapp.azure.com"]
+    certificate_name = "wildcard-ithc-platform-hmcts-net"
+    global_exclusions = [
```

do you need this for a testing app? I would just turn off prevention mode
```suggestion
	mode              = "Detection"
    global_exclusions = [
```